tests/models/test_player.py

```diff
@@ -0,0 +1,23 @@
+"""A module for testing player model."""
+import pytest
+from django.contrib.auth.models import User
+
+from api.models import Player
+
+pytestmark = pytest.mark.django_db
```

Comment:
this one's used when instead of duplicating the decorator *multiple* times usually